target/linux/mvebu/config-4.9

```diff
@@ -60,6 +60,8 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
+CONFIG_BTRFS_FS=y
```

Comment:
This is not raw NAND where UBIFS should be used, eMMC devices have a flash translation layer. F2FS may be optimised for this case, but that does not really make Btrfs (or ext4) a bad idea.

I would prefer to reach some consensus on this matter before doing any further work towards the current (bloat-inducing) medkit/Btrfs approach or alternative (and more complicated for end users) TFTP flash, u-boot environment update, ClearFog-like sysupgrade approach.